BUILD.md

```diff
@@ -142,6 +142,23 @@ bin/linkerd -n emojivoto stat deployments
 bin/linkerd -n emojivoto tap deploy voting
 ```
 
+#### Deploying Control Plane components with Tracing
+
+Control Plane components have `trace-collector`, `sampling-probability` flags used to enable [Distributed Tracing](https://opentracing.io/docs/overview/what-is-tracing/) for development purposes. It can be enabled globally i.e Control plane components and their proxies by using the `--control-plane-tracing` installation flag.
+
+This will configure all the components to send the traces at `linkerd-collector.linkerd-tracing:55678`
+
+```bash
+
+# install Linkerd with tracing
+linkerd install --control-plane-tracing | kubectl apply -f -
+
+# install OpenCensus collector and Jaeger collector to collect traces
+kubectl apply -f kubectl apply -f https://gist.githubusercontent.com/Pothulapati/245842ce7f319e8bcd02521460684d6f/raw/39bacaa657255eca59de768fefef10da2835e966/linkerd-tracing.yaml
```

Comment:
should one of these `kubectl apply -f` be `linkerd inject`?

Reply:
as the namespace has the inject annotation already. `linkerd inject` is not needed. sorry